pkg/katautils/config.go

```diff
@@ -120,11 +121,12 @@ type proxy struct {
 }
 
 type runtime struct {
-	Debug               bool   `toml:"enable_debug"`
-	Tracing             bool   `toml:"enable_tracing"`
-	DisableNewNetNs     bool   `toml:"disable_new_netns"`
-	DisableGuestSeccomp bool   `toml:"disable_guest_seccomp"`
-	InterNetworkModel   string `toml:"internetworking_model"`
+	Debug               bool     `toml:"enable_debug"`
+	Tracing             bool     `toml:"enable_tracing"`
+	DisableNewNetNs     bool     `toml:"disable_new_netns"`
+	DisableGuestSeccomp bool     `toml:"disable_guest_seccomp"`
+	Experimental        []string `toml:"experimental"`
```

Comment:
Could you bump `formatVersion` to indicate a format change (if there are experimental features?

Reply:
Oh, I missed that. Adding...